machines/Issuers/IssuersMachine.ts

```diff
@@ -394,14 +394,18 @@ export const IssuersMachine = model.createMachine(
           src: 'verifyCredential',
           onDone: [
             {
-              actions: ['sendSuccessEndEvent', 'setIsVerified'],
+              actions: ['sendSuccessEndEvent', 'setIsVerified', 'setIsExpired'],
```

Comment:
no need to have separate action.
isExpired Vc part of vc verification, so should have single action which makes VC verified and expired if any.

Reply:
Done